siliconcompiler/tools/openroad/show.py

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
openroad sc_apr.tcl is that is that using step or task name

Reply:
It still has `set sc_task   $sc_step` - I'll fix that.